Rate Counter/readme.md

```diff
@@ -0,0 +1,29 @@
+# Rate Counter
+
+Design a rate counter to get the total count in a time range.
```

Comment:
There's not enough context here for me to understand what is going on.

Reply:
Will add more context.